hack/release/prepare-changelog.sh

```diff
@@ -25,6 +25,10 @@ Draft the changelog for a release based on PR history starting from the last rel
 For a minor release such as 1.5.0, the changelog will base on PRs merged to main branch since the fork of release-1.4.
 For a patch release such as 1.5.2, the changelog will base on PRs merged to release-1.5 branch since the release of 1.5.1.
 
+Please make sure you check out the release branch before running this script for a patch release. E.g.:
+#git checkout --track upstream/release-1.5
+#./hack/release/prepare-changelog.sh --release 1.5.2
```

Comment:
The script could fetch upstream and use upstream branch directly, instead of requiring user to prepare it manually.

Reply:
Updated.